### PR TITLE
Allow multiple queries on full-text columns.

### PR DESCRIPTION
**Pull request by jmcarp**
Permit queries like `contributor_name=smith&contributor_name=jones`.
Required for https://github.com/18F/openFEC-web-app/issues/1022.

cc @LindsayYoung

### CODE REVIEW

**Comment by jmcarp** [comment]
Note: this patch modifies full-text search on contributor name (receipts) and recipient name (disbursements) to allow for searching on both name and committee ID. If we decide that *all* filter typeaheads should support search on both ID and text, we'll need to make this change to a few more columns.

cc @noahmanger @LindsayYoung @jenniferthibault

**Comment by jmcarp** [comment]
This is part of the consolidated typeahead feature. Assigning to @ccostino for review.

File: webservices/resources/candidates.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class CandidateList(ApiResource):
     schema = schemas.CandidateSchema
     page_schema = schemas.CandidatePageSchema
     filter_multi_fields = filter_multi_fields(models.Candidate)
+    filter_fulltext_fields = [('q', models.CandidateSearch.fulltxt)]
```
Comment (ccostino) [inline webservices/resources/candidates.py]: What specifically is going on here?  It looks like this is just setting a property on the class that is a list value comprising of tuples that represent each field, but I just want to be sure.  What does the `'q'` in the tuple represent in this case?

File: webservices/args.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ def make_seek_args(field=fields.Int, description=None):
     }
 
 names = {
-    'q': fields.Str(required=True, description='Name (candidate or committee) to search for'),
+    'q': fields.List(fields.Str, required=True, description='Name (candidate or committee) to search for'),
```
Comment (ccostino) [inline webservices/args.py]: Just checking to make sure my assumption is correct on these changes - this is being changed from accepting a single value for a key to being able to support multiple values for the same key?

**Comment by ccostino** [comment]
@jmcarp:  I only had a couple of questions more for just verifying my own understanding, otherwise this looks good to merge.  Thanks!

File: webservices/args.py (hunk 2 above)
Comment (jmcarp) [inline webservices/args.py]: :+1:

File: webservices/resources/candidates.py (hunk 1 above)
Comment (jmcarp) [inline webservices/resources/candidates.py]: For implementation, check out `ApiResource` in `webservices/common/views.py`. This particular class variable is a list of tuples relating query parameters (in this case, "q") to tsvector columns for use in filtering. Basically, DRF-lite.

Comment (ccostino) [inline webservices/resources/candidates.py]: Okay, I see now, thank you!

**Pull request merged**